### PR TITLE
Docs

### LINKED ISSUE #15: Documentation

### LINKED ISSUE #26: User Guide

### LINKED ISSUE #66: Document Libraries
Forgot to add the [library contracts](https://github.com/1Hive/token-request-app/tree/master/contracts) to the docs. Since we built them ourselves and they're not imported they will need to be audited, and thus documented.

### PR DESCRIPTION
**Pull request by yeqbfgxjiq**
closes #15, closes #26, closes #66

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,4 +65,22 @@ npm run start:template
 
 ## How to deploy to an organization
 
-TODO:
+Token Request app has been deployed to Rinkeby at `token-request.open.aragonpm.eth`
```
Comment (rperez89) [inline README.md:68]: token-request-staging.open.aragonpm.eth?

File: docs/installation-guide.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+This guide will walk you through everything you need to add Token Request to an existing Aragon DAO.
+
+---
+
+## TL;DR
+
+Token Request has been published to the following locations:
+
+- Rinkeby: `token-request.open.aragonpm.eth`
```
Comment (rperez89) [inline docs/installation-guide.md:11]: same here

File: docs/installation-guide.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+## Detailed Installation Guide
+
+---
+
+### 1. Deploy a fresh DAO
+
+This step is if you don't already have a DAO to install Token Request on, or want to test it on a demo first.
+
+First, make sure that you have the [Aragon CLI](https://hack.aragon.org/docs/cli-intro.html) installed. Then run `aragon devchain` in a terminal. This should show you two Ethereum addresses. The first one has the most permissions and is used to execute commands on the Aragon CLI. Import the private key of that account into Metamask. Then head over to the [Rinkeby DAO launcher](rinkeby.aragon.org) and create a DAO with the democracy kit. Make sure that the Metamask account that is active is the first account created by your `aragon devchain`.
```
Comment (rperez89) [inline docs/installation-guide.md]: democracy kit?

File: docs/installation-guide.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+Once your Democracy DAO is deployed (the voting params don't matter as you'll be the only one voting right now), go to the settings tab where you will find the addresses for the DAO and its apps. For legibility of subsequent commands will set bash environment variable for these addresses:
```
Comment (rperez89) [inline docs/installation-guide.md:34]: same here

File: docs/technical-guide.md [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+## User Defined Global Variables
+
+These variables are available in the global scope of the contract, but can be changed via the contract's functions.
+
+- `tokenManager` is an Aragon [Token Manager](https://wiki.aragon.org/dev/apps/token-manager/)
+- `vault` is an Aragon [Vault](https://wiki.aragon.org/dev/apps/vault/)
+- `acceptedDepositTokens` is a dynamically sized array that holds the addresses of token contracts
+- `nextTokenRequestId` is the id that the next token request created will have.
+- `tokenRequests` is a mapping between a `uint256` and a `TokenRequest` struct. This holds all requests ever created.
+- `addressesTokenRequestIds` is a mapping of a user's address to a dynamically sized array of `uint256` where each `uint256` is mapped to a TokenRequest in `tokenRequests`.
```
Comment (rperez89) [inline docs/technical-guide.md]: this variable does not exist anymore

File: docs/technical-guide.md [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+
+```
+TokenManager public tokenManager;
+address public vault;
+
+address[] public acceptedDepositTokens;
+
+uint256 public nextTokenRequestId;
+mapping(uint256 => TokenRequest) public tokenRequests; // ID => TokenRequest
+mapping(address => uint256[]) public addressesTokenRequestIds; // Sender address => List of ID's
```
Comment (rperez89) [inline docs/technical-guide.md]: we should remove from here also

**Review by rperez89 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fabriziovigevani) [reply README.md:68]: We will most likely have all apps deployed pretty soon with the regular name (without staging)

File: docs/installation-guide.md (hunk 3 above)
Comment (fabriziovigevani) [reply docs/installation-guide.md]: Will rename to template

**Review by rperez89 — APPROVED** [review]

**Pull request merged**